packages/core-utils/src/query-params.js

```diff
@@ -1,3 +1,7 @@
+// TODO: Remove this entire file, as it is deprecated in favor of i18n-queryParams within
```

Comment:
Why don't we move the deprecated contents to `deprecated.js`?

Reply:
There's an upcoming PR that just deletes this file so I didn't see the value in moving everything over